README.md

```diff
@@ -1,2 +1,2 @@
-# Go Rails
+
```

Comment:
Why do you add this empty line? Maybe you could remove it if needed

Reply:
Thank you, Master!

Reply:
*if it's needed
It's passive voice in Present SImple